packages/website/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["plugin:react/recommended"],
+  "rules": {
+    "react/prop-types": "off",
+    "react/no-unescaped-entities": "off"
```

Comment:
Since this is a Gatsby site, it will be common to have text, and so having to escape double quotes and single quotes etc. is a big pita. 

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

My theory is that it makes a lot of sense for code, but not so much for content sites?